composer.json

```diff
@@ -19,5 +19,13 @@
         "psr-4": {
             "PHPUnitRetry\\": "src/"
         }
+    },
+    "autoload-dev": {
```

Comment:
I'm not sure why the tests were not registered

Reply:
So, what is the purpose of autoloading test classes when that is handled by `phpunit`? It would make sense if I had helper classes to autoload, but since that's not the case, what is the point? Is it just a convention for developers to see where the library tests are?

Reply:
It's indeed more of a convention than necessity in this case. That said as soon as you have a test class (as a fixture) in its own file or other testing tools like Behat, it becomes a necessity. Since the cost of including the `autoload-dev` is close to non-existent, the practice is to usually just have it and not have to worry about when it actually becomes a necessity